src/Microsoft.AspNetCore.Mvc.Abstractions/Formatters/IFormatterExceptionPolicy.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Formatters
+{
+    /// <summary>
+    /// A policy which <see cref="IInputFormatter"/>s can implement to indicate if they want the body model binder
+    /// to handle all exceptions. By default, all default <see cref="IInputFormatter"/>s implement this interface and
+    /// have a default value of <see langword="true"/>.
+    /// </summary>
+    public interface IFormatterExceptionPolicy
+    {
+        /// <summary>
+        /// Gets the flag to indicate if the body model binder should handle all exceptions. If an exception is handled,
```

Comment:
> Gets a value that determines if the `<see cref="BodyModelBinder" /> `

Reply:
Yeah I tired that earlier, but since this is in 'abstractions', I cannot reference the binder as its in 'Core'